src/geometry/coordinate_systems.hpp

```diff
@@ -53,6 +53,7 @@ namespace Geometry {
 // Coordinate system choices
 using CoordSysMeshBlock = GEOMETRY_MESH_BLOCK;
 using CoordSysMesh = GEOMETRY_MESH;
+constexpr char GEOMETRY_NAME[] = TOKEN2STRING(PHOEBUS_GEOMETRY);
```

Comment:
Aside, the existence of this thing means that in the places where we do
```C++
if (typeid(PHOEBUS_GEOMETRY) == typeid(THING))
```
we can instead do
```C++
if constexpr(constexpr_strings_equal(Geometry::GEOMETRY_NAME, "Geometry::XX"))
```
if we implement something like
```C++
#include <string_view>

constexpr bool constexpr_strings_equal(char const * a, char const * b) {
    return std::string_view(a)==b;
}
```
if we're feeling spicy and upgrade to C++17

Reply:
True, something to keep in mind if we revamp geometry in the future (maybe to make it a template parameter so we dont have to recompile everything to get a new geometry)